extensions/jwks/src/main/java/de/sovity/edc/extension/jwks/controller/JwksJsonTransformer.java

```diff
@@ -0,0 +1,7 @@
+package de.sovity.edc.extension.jwks.controller;
+
+import com.nimbusds.jose.jwk.JWK;
+
+public interface JwksJsonTransformer {
```

Comment:
Violates YAGNI. We only have one implementation, why do we need this?

Reply:
Because it enables the D of SOLID: https://www.baeldung.com/solid-principles#d

Reply:
Clean code still remains, no need for abstractions when there is no need.

Dependency Inversion requires a need for abstraction in the first place.

Careful with bringing too many OOP-related ideas into Web Development. Keeping OOP to a minimum is a requirement for code with low mental complexity. For OOP, which should be kept to libaries, data stractures, algorthims, etc., there are many more crazier ways to write and organize code, which are all not worth it to "spend" on simple things. Let's not go through them one by one please, let's keep it short and simple.